packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -98,6 +98,10 @@ contract L2OutputOracle is Ownable {
         uint256 _startingBlockNumber,
         address sequencer
     ) {
+        require(
+            _startingBlockNumber > _historicalTotalBlocks,
```

Comment:
`0 > 0 => false`. This does bring up some interesting questions around the L2 genesis - is it a block that actually exists or not and what relation is there between created blocks & that block

Reply:
Right, my sense is that our testing is currently assuming that both of these numbers are zero.

Reply:
I've removed this commit for now to get this PR through.